instat/dlgFactorLabels.Designer.vb

```diff
@@ -57,10 +57,10 @@ Partial Class dlgViewFactorLabels
         Me.grpSummaryStatistics.Controls.Add(Me.ucrChkShowPercentage)
         Me.grpSummaryStatistics.Controls.Add(Me.ucrChkShowFrequencies)
         Me.grpSummaryStatistics.Controls.Add(Me.ucrChkShowMissingValues)
-        Me.grpSummaryStatistics.Location = New System.Drawing.Point(316, 199)
+        Me.grpSummaryStatistics.Location = New System.Drawing.Point(170, 199)
         Me.grpSummaryStatistics.Name = "grpSummaryStatistics"
-        Me.grpSummaryStatistics.Size = New System.Drawing.Size(140, 90)
-        Me.grpSummaryStatistics.TabIndex = 5
+        Me.grpSummaryStatistics.Size = New System.Drawing.Size(154, 90)
+        Me.grpSummaryStatistics.TabIndex = 4

```

Comment:
I moved the Summary Statistics groupbox as I feel it should be next to the Variable Labels groupbox as these are both about the data. Whereas the final groupbox is about layout. This meant changing the tab order.